src/modules/library/containers/Import.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+	CardBody, Row, Col,
+	Button, Input, Label,
+	FormGroup
+} from 'reactstrap';
+
+const Import = ({
+	api, app, setChosenPanel, retrieveAll
+}) => {
+	const { t } = useTranslation();
+	const [chosenFilename, setChosenFilename] = useState("");
+	const [type, setType] = useState("override");
+	const [errors, setErrors] = useState("");
+	const inputFileRef = useRef(null)
+	const formRef = useRef(null);
+
+	const chooseFile = () => {
+		if (!inputFileRef.current) return;
+
+		inputFileRef.current.click();
+	}
+
+	const updateFilename = () => {
+		if (!inputFileRef.current) return;
+
+		// Get filename from input and remove path
+		const filename = inputFileRef.current.value.replace(/.*[\/\\]/, '');
+		setChosenFilename(filename);
+
+		// Check if file is tar
+		if (!filename.includes(".tar")) {
+			setErrors(true);
+		} else {
+			setErrors(false);
+		}
+	}
+
+	const onTypeChange = (e) => setType(e.target.value);
+
+	const importLibrary = async event => {
+		event.preventDefault();
+		try {
+			const data = new FormData(formRef.current);
+			const response = await api.put(`/library/upload?type=${type}`, data, {
+				headers: {
+					'Content-Type': 'multipart/form-data'
+				}
+			})
+
+			if (response.data.result !== "OK") throw new Error(`Response result is ${response.data.result}. Expected result is OK`);
+
+			setChosenPanel("editor");
+			retrieveAll();
+			app.addAlert("success", t("ASABLibraryModule|Library has been successfully uploaded"));
+		} catch (e) {
+			console.error("Failed to upload library\n", e);
+			app.addAlert("warning", t("ASABLibraryModule|Failed to upload library"));
+		}
+		
+	}
+
+	return (
+		<CardBody>
+			<form
+				id="upload-library"
+				ref={formRef}
+				onSubmit={importLibrary}
+			>
+				<Col>
+
+					<Row>
+						<Col>
+							<h5>{t("ASABLibraryModule|Import library")}</h5>
+							<p>{t("ASABLibraryModule|Choose library you want to import (only tar files are allowed) and type of importing")}</p>
```

Comment:
Can it be as following?

```
<p>{t("ASABLibraryModule|Choose library and type for import")}</p>
<p style={{color: "#c8ced3", size: "8px"}}>{t("ASABLibraryModule|Only tar files are allowed")}</p>
```
That `Only tar files are allowed` looks like a hint (similar to FormText)